lxd/instance/drivers/driver_lxc.go

```diff
@@ -8419,7 +8422,7 @@ func (d *lxc) Metrics(hostInterfaces []net.Interface) (*metrics.MetricSet, error
 
 	// Get CPU stats
 	usage, err := cg.GetCPUAcctUsageAll()
-	if err != nil {
+	if err != nil && isRunning {
```

Comment:
If the instance isn't running what would be in the `lxd_cpu_seconds_total` metric? I'm assuming we wouldn't know there are `system` and `user` modes to report as 0. Or maybe I'm just missing something here :/

Reply:
You're right. Some memory stats, CPU seconds, `lxd_cpu_effective_total`, and disk IO unfortunately won't be listed.